android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/Settings.kt

```diff
@@ -80,6 +80,32 @@ abstract class Settings {
      */
     open var cookieBannerHandlingGlobalRulesSubFrames: Boolean by UnsupportedSetting()
 
+    /**
+     * Setting to control the cookie banner enables / disables the URL query string
+     * stripping in normal browsing mode which strips query parameters from loading
+     * URIs to prevent bounce (redirect) tracking.
+     */
+    open var queryParameterStripping: Boolean by UnsupportedSetting()
+
+    /**
+     * Setting to control the cookie banner enables / disables the URL query string
+     * stripping in private browsing mode which strips query parameters from loading
+     * URIs to prevent bounce (redirect) tracking.
+     */
+    open var queryParameterStrippingPrivateBrowsing: Boolean by UnsupportedSetting()
+
+    /**
+     * Setting to control the list which contains sites where should
+     * exempt from query stripping.
```

Comment:
Setting to control the list that contains sites where they should
exempt from query stripping.